www/templates/jsontestresults.html

```diff
@@ -27,7 +27,7 @@
                 <div class="clearfix"></div>
                 {% for artifact in suite.artifacts %}
                     <div class="artifact">
-                        <a target="_blank" href="{{ join(path_to_artifacts, artifact) }}">{{ basename(artifact) }}</a>
+                        <a target="_blank" href="{{ join(path_to_artifacts, artifact).replace("\\","/") }}">{{ basename(artifact.replace("\\","/")) }}</a>
```

Comment:
You might want to use `urlparse.urljoin` instead.

That is assuming that we can pass `urljoin` to this (jinja?) template, and that `path_to_artifacts` does not contain `\` already :)